### ISSUE TITLE
Update README for Clarity

### ISSUE THREAD

**Issue by rdevathu** [ComputationalMedicineLab/fast_intensity#10]
Please update the instructions in the read-me to use pip3 instead of pip. In some cases pip and pip3 may not be soft linked which will lead to the following error: 

 `ValueError: 'fast_intensity.pyx' doesn't match any files` 

This is because pip from Python 2 is used and it seems that file path conventions in setup.py may have changed between Python 2 and Python 3. This helps reduce confusion during installation.

**Comment by johnstill** [comment]
Whether `pip` is derived from Python 2 or 3 is platform / distribution dependent; `pip --version` should tell you which python is involved, and a safe way to make sure install to the correct python's site-directories is to invoke pip via python, e.g. as `python -m pip install ...`

At any rate I don't believe we support Python 2 at all, which I will be happy to specify in the Readme.  I'll also add a blurb warning users that they may need to check which python `pip` is associated with.

**Comment by johnstill** [comment]
Ok - I dug into this issue a little bit further and realized it is not a Python 2 vs 3 thing, its a bug in the distribution of the source tarball (it literally just doesn't include the `.pyx` file).  So my guess is that you wound up installing the wheel, if you got it installed at all.  

This will be fixed when we release version 0.4; until then, you can either make sure you're installing the wheel, or install from the github repository directly.

**Comment by rdevathu** [comment]
Oh, interesting. I'm glad you were able to find the root issue I'm sure it will help in future installations. The reason I assumed that was because of this other issue report on Git. Link: https://github.com/flatironinstitute/online_psp/issues/11 

The developer responded that he changed the file path and so I assumed that perhaps you needed the absolute path instead of the relative path to the .pyx file. From here I downloaded the project from Git changed the file path in the setup.py file and used pip on the local directory. This dropped the error that said the file could not be found but presented a new error with the use of the open() function since "encoding = 'UTF-8'" was not supported in Python2. This is when I realized I should use pip3 but then interestingly the previous error reappeared. When I finally changed the file path back to the "relative path" (same as that in version .3) pip3 was able to do its job. 

In the end, it appears that I was able to install it in my venv using pip3 after downloading from Github. <del>Also just a quick clarification. At the bottom of your setup.py it states that a required dependency is scipy but this is not included your install instructions.</del> On second inspection it appears that you have updated this or I missed it.
